Fire `get_started_faq` tracking event

tomalec opened the pull request:
Addresses a part of https://github.com/woocommerce/pinterest-for-woocommerce/issues/232

<!-- You can erase any parts of this template not applicable to your pull request. -->

<!-- Reference issue(s) that this PR fixes or addresses -->

### Changes proposed in this pull request
<!-- Explain what this PR adds or changes, why, and how this will impact users. -->

When clicking on getting started page faq item to collapse or expand it.
Fire `get_started_faq` tracking event,
Install `@woocommerce/tracks` locally, for linting and testing. It's externalized with DEWP.

#### Screenshots
<!--- Optional --->
![track](https://user-images.githubusercontent.com/17435/140065093-5689a1f2-6283-4ae2-92d6-6327cc7356e1.gif)


### Detailed test instructions
<!-- Add steps to confirm the fix or change. -->
0. Enable logging tracking events in your browser `localStorage.setItem('debug', 'wc-admin*')`
1. Visit getting started page [`/wp-admin/admin.php?page=wc-admin&path=%2Fpinterest%2Flanding`](https://wp581wc580.test/wp-admin/admin.php?page=wc-admin&path=%2Fpinterest%2Flanding)
1. Expand and collapse the FAQ questions

<!-- Please add details of other areas that might be impacted by the change. -->

### Additional notes:
1. The docs `.md` will be added in a separate PR with JSDoc automation.
2. I do not add a specific changelog note, as I'd rather add a single entry when merging the entire feature branch.
3. Sorry for the unfortunate branch name :( this PR's branch should be called like `add/232-faq-tracking`)

### Changelog note

tomalec committed:
Fire `get_started_faq` tracking event,

When clicking on getting started page faq item to collapse or expand it.

Install `@woocommerce/tracks` locally, for linting and testing. It's externalized with DEWP.

Addresses a part of https://github.com/woocommerce/pinterest-for-woocommerce/issues/232

tomalec committed:
Provide `question_id` to `wcadmin_pfw_get_started_faq` event,

Adhere to Tracks Event naming convention.

tomalec committed:
Document the `wcadmin_pfw_get_started_faq` event.

eason9487 commented on assets/source/setup-guide/app/views/LandingPageApp.js:
```diff
@@ -187,9 +190,42 @@ const FaqSection = () => {
+const FaqQuestion = ( { questionId, question, answer } ) => {
+	const panelToggled = useCallback(
+		( isOpened ) => {
+			recordEvent( 'wcadmin_pfw_get_started_faq', {
```
❔ 
It shows the event name `wcadmin_wcadmin_pfw_get_started_faq` on DevTool console. Are the repeated prefixes `wcadmin_` expected?

tomalec replied:
Thanks for catching it. :) It's not expected.
I got too attached to FieldGuide's naming convention rule:
> Don’t programmatically create event names. They should be hardcoded so they are easily searchable in the code base.

And `@woocommerce/tracks` already concatentates `wcadmin_`

eason9487 reviewed (approved):
Tested locally and the tracking events are triggered. LGTM. Left a comment about the event name, but even if it needs adjustment, should be no need to review again.

tomalec committed:
Remove duplicated `wcadmin_` from event name,

addresses https://github.com/woocommerce/pinterest-for-woocommerce/pull/237#discussion_r742623832.
